Content.Server/Explosion/EntitySystems/TwoStageTriggerSystem.cs

```diff
@@ -60,11 +61,13 @@ public override void Update(float frameTime)
             if (component.NextTriggerTime == null)
                 continue;
 
+            if (component.Triggered && !component.ComponentsIsLoaded)
+                LoadComponents(uid, component);
```

Comment:
Since this is meant to be run when the component is triggered the first time I would recommend moving it into `OnTriggered`.

Reply:
Honestly not certain why it's in Update, but it probably shouldn't be there.

Reply:
Oh, and this move LoadingComponents after all triggers, that is better, and is not removing current components that are going to be triggered